presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationJdbcSmokeTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.testing.MaterializedResult;
+import org.testng.annotations.Test;
+
+import java.sql.SQLException;
+
+import static com.facebook.presto.testing.assertions.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+
+public abstract class AbstractTestIntegrationJdbcSmokeTest
+        extends AbstractTestIntegrationSmokeTest
+{
+    protected AbstractTestIntegrationJdbcSmokeTest(QueryRunnerSupplier supplier)
+    {
+        super(supplier);
+    }
+
+    @Test
+    public void testShowCreateTable()
+            throws SQLException
```

Comment:
Why not to add these tests to `AbstractTestIntegrationSmokeTest` and disable them for other connectors?

Reply:
What is the mechanism to disable for other connectors?  Is it overriding with a noop?